main.py

```diff
@@ -1,7 +1,22 @@
 import asyncio
 from src.scrapers.colleges import careers360
+from src.scrapers.graphy.assignments import GraphyAssignmentScraper
+import logging
+
+scraper = 2
 
 if __name__ == "__main__":
-    start_page = int(input("Enter start page: ").strip())
-    end_page = int(input("Enter end page: ").strip())
-    asyncio.run(careers360.main(start_page=start_page, end_page=end_page))
+    if scraper == 1 :
+        start_page = int(input("Enter start page: ").strip())
+        end_page = int(input("Enter end page: ").strip())
+        asyncio.run(careers360.main(start_page=start_page, end_page=end_page))
+    else:
+        email = input("Enter Email Address: ").strip()
+        password = input("Enter Password: ").strip()
```

Comment:
Can you take all these inputs from a config file in `.cfg` or `.yml` format?
`input()` defeats the purpose of automation, when we want to combine this script with something else.